src/main/java/Storage.java

```diff
@@ -0,0 +1,80 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Storage class serves to save users' task lists to local machines' disk
+ * so that the previous run's list will be available to users for reference.
+ */
+public class Storage {
+
+    //public static String home = System.getProperty("src/data/duke.txt");
+    private static final String home = "data";
+
+    private static java.nio.file.Path path = java.nio.file.Paths.get(home, "duke.txt");
+
+    private static boolean directoryExists = java.nio.file.Files.exists(path);
+
+    public static void read() {
+        try {
+            File storage = new File(home);
+
+            if(!directoryExists){
```

Comment:
Would it better adhere to coding standards if you added white spaces before the "(" and after the ")" ? This style appears quite often in your code so hopefully you take a look at it again.